pkgs/development/python-modules/docling-core/default.nix

```diff
@@ -2,6 +2,7 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  fetchpatch,
```

Comment:
```suggestion
```
Apparently unused.

Reply:
*sigh* AuDHD, man. Makes it easy to forget I didn't go the `fetchpatch` route after all.

Fixed.